src/lib.rs

```diff
@@ -136,74 +136,101 @@ pub fn initialize(_argc: *const isize, _argv: *const *const *const u8) -> isize
 #[macro_export]
 macro_rules! fuzz_target {
     (|$bytes:ident| $body:block) => {
-        /// Auto-generated function
-        #[no_mangle]
-        pub extern "C" fn rust_fuzzer_test_input($bytes: &[u8]) {
-            // When `RUST_LIBFUZZER_DEBUG_PATH` is set, write the debug
-            // formatting of the input to that file. This is only intended for
-            // `cargo fuzz`'s use!
+        const _: () = {
```

Comment:
Isn't this unnecessary for `macro_rules!` macros, since they have hygiene, unlike proc macros?

Reply:
AFAIK they're only hygienic in terms of local variable names within functions, for items defined within a module there's no hygiene and it's just the "copy and paste" approach.

Reply:
It seems you're right! Never knew that: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=8abc44430951552a52ef9cff4e3ec8ba